core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -251,20 +255,46 @@ public Response free(@QueryParam("path") String path,
   }
 
   /**
-   * @summary get the file descriptors for a path
+   * @summary get the file descriptors for a path. This is deprecated since 1.1.1 and will be
+   * removed in 2.0. Use LIST_STATUS2.
    * @param path the file path
    * @param loadDirectChildren whether to load direct children of path
    * @return the response object
+   * @deprecated since 1.1.1 and will be removed in 2.0
    */
   @GET
   @Path(LIST_STATUS)
   @ReturnType("java.util.List<alluxio.wire.FileInfo>")
+  @Deprecated
```

Comment:
adding a param here is backwards compatible, so no need to create a new endpoint